### ISSUE TITLE
No Face Founded in some images

### ISSUE THREAD

**Issue by seebrand** [ageitgey/face_recognition#572]
* face_recognition version:1.22,model:0.30,dlib:19.8.1
* Python version:3.6
* Operating System:windows 7

### Description

Most go well, but some cam images cannot detect face. original blur image after de-bluring, can find face  by CV2, but    face_recognition cannot find any face:

image = face_recognition.load_image_file("id-err.jpg")

face_locations = face_recognition.face_locations(image)

print("I found {} face(s) in this photograph.".format(len(face_locations)))
# return 0， no find face.
image_to_test = face_recognition.load_image_file("id-err.jpg")

#error happended 。
image_to_test_encoding = face_recognition.face_encodings(image_to_test)[0]  
....

![id-err](https://user-images.githubusercontent.com/32126008/43001971-c187f44c-8c59-11e8-9cd1-ddfbc0bfe5f3.jpg)
...

By CV2,can detect face:

    gray = cv2.cvtColor(frame, cv2.COLOR_BGR2GRAY)                 
    #

    faceRects = classfier.detectMultiScale(gray, scaleFactor = 1.2, minNeighbors = 3, minSize = (32, 32))
    if len(faceRects) > 0:                                     
        for faceRect in faceRects:  
            x, y, w, h = faceRect      
            print("face: %s,%s,%s,%s"%(x, y, w, h))
...

![id-err jpg iddet](https://user-images.githubusercontent.com/32126008/43002012-e62e575a-8c59-11e8-9271-a75384b74977.png)

**Comment by fawkesguii** [comment]
I am a newbie in the field, and I am using lib for studies and in a project of the company that I work for. I've been through something like that.

In my case it was the size of the image, I use mxnet apache to do the detection of the faces and from them generate the encoding of the face for recognition.
In some of my cases the face image was very small, just increasing the image I was able to recognize the encoding (consequently you will be able to find the location of the face).

I took his picture and circled the face_locations using the standard model and the cnn model.
With the standard model I had to resize the image to locate the face, while in the cnn model it was not necessary.

I will not know to explain the reason to you, since I am of the non-theoretical part :/
But I hope I have helped with something.q
![capturar1](https://user-images.githubusercontent.com/18038770/43590114-a7be2864-9646-11e8-8451-b5b8617b0450.JPG)
---
![capturar](https://user-images.githubusercontent.com/18038770/43590115-a7e66ab8-9646-11e8-81e7-4d8fbcd12d44.JPG)


Sorry if I wrote something wrong, I do not speak English.
